test-app/Xliff-test/src/TestCustomerApi.Page.al

```diff
@@ -0,0 +1,30 @@
+page 50002 "Test Customer Api"
+{
+    PageType = API;
+    Caption = 'My Customer API';
+    APIPublisher = 'contoso';
+    APIGroup = 'app1';
+    APIVersion = 'v2.0', 'v1.0';
```

Comment:
Aaah, didn't know that an API page could have multiple versions! 👍

Reply:
Neither did I, copy, paste from docs :laughing: